apps/backoffice-v2/src/domains/business-reports/adapters/report-adapter/report-adapter.ts

```diff
@@ -17,10 +19,20 @@ const toRiskLabels = (riskIndicators: Array<{ name: string; riskLevel: string }>
 
   return riskIndicators.map(({ name, riskLevel }) => ({
     label: name,
-    severity: riskLevel,
+    severity:
+      severityToDisplaySeverity[riskLevel as keyof typeof severityToDisplaySeverity] ?? riskLevel,
   }));
 };
 
+const normalizeRiskLevel = (riskTypeLevels: Record<string, TSeverity>) => {
```

Comment:
There might be a better name for the function. It does something more specific than normalizing.